src/test/js/reset_password_test.js

```diff
@@ -32,6 +36,10 @@ BeforeSuite(async ({ I }) => {
 
     await I.createUserWithRoles(citizenEmail, userPassword, randomUserFirstName + 'Citizen', ["citizen"]);
     userFirstNames.push(randomUserFirstName + 'Citizen');
+    await I.createUserWithRoles(diffCaseCitizenEmail, userPassword, randomUserFirstName + 'diffcase', ["citizen"]);
```

Comment:
How do we delete test those users?

Reply:
All users in the test have the same prefix, so we're using testDataPrefix to delete the data. 
e.g. testing-support/test-data?async=false&testDataPrefix=SIDM_TEST_GJHMni

Reply:
How is the prefix added to the username?